yarn.lock

```diff
@@ -7170,6 +7170,15 @@ http-errors@~1.7.2:
     statuses ">= 1.5.0 < 2"
     toidentifier "1.0.0"
 
+http-proxy-agent@^4.0.1:
```

Comment:
this was autogenerated by `yarn`, probably because we forgot to reinstall deps after resolving merge conflicts